pybricksdev/ble.py

```diff
@@ -9,16 +9,16 @@
 from bleak.backends.scanner import AdvertisementData
 
 
-async def find_device(name: str, timeout: float = 5) -> BLEDevice:
-    """Quickly find BLE device address by friendly device name.
+async def find_device(device_or_address: str, timeout: float = 5) -> BLEDevice:
+    """Quickly find BLE device address by friendly device name or address.
```

Comment:
We should also mention that "address" does not work on macOS. Instead, macOS assigns each device a UUID that has to be used instead of the Bluetooth address.